docs/docs/ref-05-events.ko-KR.md

```diff
@@ -139,7 +139,6 @@ DOMEventTarget relatedTarget
 
 이 포커스 이벤트는 폼 엘리먼트뿐만 아니라 모든 React DOM 엘리먼트에서 작동합니다.
 
-<a name="form-events"></a>
```

Comment:
You can read more at [here](https://github.com/gettalong/kramdown/commit/6d45c62941ce97376d390c3df9374126d5e0972e).